test/actions/FilterActionsTest.js

```diff
@@ -23,9 +23,8 @@ describe('FilterActions', function () {
   describe('clearFilterIncidentId', function(){
     const filter = {incidentId: 1, eventTypes: [1, 2]}
     const result = filterActions.clearFilterIncidentId(filter)
-    const expectedValue = {type: 'CLEAR_EVENT_FILTER_INCIDENTID', filter: {incidentId: 1, eventTypes: [1, 2]}}
     it('should return an object with a type and a filter', function(){
```

Comment:
Let's keep the 'should' statements in line with what's actually being checked. In this case, that might look like 'should return an object with no incidentId value'.

Reply:
The FilterActionTest will not return a cleared filter, it will only return an action with the type CLEAR_eVENT_FILTER_INCIDENT.

Reply:
Ah, right, in that case it should return an object with the correct type and no other properties.